.github/workflows/tos.yaml

```
@@ -16,8 +16,25 @@ env:
   GH_REPO: ${{ github.repository }}
 
 jobs:
+  react-to-new-issue:
+    if: github.event_name == 'issues' && github.event.action == 'opened'
+    runs-on: ubuntu-latest
+    outputs:
+      comment-id: ${{ steps.comment.outputs.comment-id }}
+    steps:
+      - name: Add comment
+        id: comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Thank you for submitting your extension.
+
+            The validation will start as soon as we have verified that you have accepted the [term of services](https://www.docker.com/legal/extensions_marketplace_developer_agreement/).
```

Should we simplify this message, and not let any doubt here that there is no further action needed at this stage : 
```suggestion
            The validation of your submission will start soon.
```
(avoid the user wondering if they missed something in the TOC approval)